codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Stream.java

```diff
@@ -44,6 +61,26 @@
     State state();
 
     /**
+     * Get the state as related to the {@link Http2LocalFlowController}.
+     */
+    FlowControlState localFlowState();
+
+    /**
+     * Set the state as related to the {@link Http2LocalFlowController}.
+     */
+    void localFlowState(FlowControlState state);
```

Comment:
@Scottmitch let's leave the mutators for now as that's consistent with what's there today.  We can continue this discussion in the properties PR.  Sound good?